src/main/kotlin/ucfs/claimant/consumer/properties/RdsProperties.kt

```diff
@@ -1,12 +1,14 @@
 package ucfs.claimant.consumer.properties
 
+import org.springframework.beans.factory.annotation.Value
 import org.springframework.boot.context.properties.ConfigurationProperties
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 
 @Configuration
 @ConfigurationProperties(prefix = "rds")
-data class RdsProperties(var caCertPath: String = "./rds-ca-2019-eu-west-2.pem",
+data class RdsProperties(@Value("rds.truststore") var truststore: String = "",
+                         @Value("rds.truststore.password") var truststorePassword: String = "",
```

Comment:
You shouldn't need the Value annotations here, if you leave them off and then set the RDS_TRUSTSTORE and RDS_TRUSTSTORE_PASSWORD env vars in https://github.com/dwp/dataworks-aws-ucfs-claimant-consumer/blob/master/ecs.tf they will get wired in.